src/components/Home.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Start from "./Start"
+import Start_Choose from "./Start Choose";
+
+
+
+function Home() {
+    return (
+        <>
+            <Start/>
```

Comment:
can we just add here content of this component?

Reply:
i've removed this component (Start) because previously there was a different concept, now it is unnecessary.